mithril-aggregator/src/commands/serve_command.rs

```diff
@@ -132,6 +133,37 @@ impl ServeCommand {
 
             Ok(())
         });
+
+        // Create a [SignerTickersImporter] only if the `cexplorer_pools_url` is provided in the
+        // config.
+        if let Some(cexplorer_pools_url) = config.cexplorer_pools_url {
+            match dependencies_builder
+                .create_signer_ticker_importer(&cexplorer_pools_url)
+                .await
+            {
+                Ok(service) => {
+                    join_set.spawn(async move {
+                        // Wait 5s to let the other services the time to start before running
+                        // the first import.
+                        tokio::time::sleep(Duration::from_secs(5)).await;
+                        service
+                            .run_forever(Duration::from_secs(
+                                // Signer Ticker Interval are in minutes
+                                config.signer_ticker_run_interval * 60,
```

Comment:
It's maybe a better idea to use seconds (as for all other delays/intervals in the conifguration)?

Reply:
This thread does not have the need to run at the second level, so enforcing minute is fine, else instead of 720 for 12 hours you would have to input 43200 (far less readable).